src/Microsoft.AspNet.OData.Shared/Formatter/ODataResourceSetItemBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
+{
+    /// <summary>
+    /// Encapsulates an <see cref="ODataResourceSetItemBase"/> .
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1012:AbstractTypesShouldNotHaveConstructors")]
+    public abstract class ODataResourceSetItemBase : ODataItemBase
```

Comment:
>ODataResourceSetItemBase [](start = 26, length = 24)

what's the usage of this class?

What's the different between ODataResourceSetItemBase as ODataItemBase?